Doxygen/doxygen_footer.html

```diff
@@ -4,7 +4,7 @@
   </div>
   <div id="copyright">
    <p>
-     &copy; 2014 Google
+     &copy; 2015 Google
```

Comment:
I am not familiar with Doxygen could this be auto generated?   Automatically pick up the current year?